clang-tools-extra/clang-move/HelperDeclRefGraph.cpp

```diff
@@ -76,9 +76,8 @@ HelperDeclRefGraph::getReachableNodes(const Decl *Root) const {
   llvm::DenseSet<const CallGraphNode *> ConnectedNodes;
   std::function<void(const CallGraphNode *)> VisitNode =
       [&](const CallGraphNode *Node) {
-        if (ConnectedNodes.count(Node))
+        if (!ConnectedNodes.insert(Node).second)
           return;
-        ConnectedNodes.insert(Node);
         for (auto It = Node->begin(), End = Node->end(); It != End; ++It)
```

Comment:
Side-note: Can use range-based for loop here.

Reply:
Sure.  I'll take care of it in a subsequent patch.